lib/happo/public/happo-styles.css

```diff
@@ -107,3 +107,26 @@ body {
 .Swiper__image {
   pointer-events: none;
 }
+
+.SideBySide__image {
+  display: inline-block;
+  line-height: 0;
+  position: relative;
+}
+
+.SideBySide__caption {
+  background-color: #666666;
+  bottom: 0;
+  color: #ffffff;
+  font-size: 10px;
+  line-height: 1;
+  opacity: 0;
+  padding: 3px 6px;
+  position: absolute;
+  right: 0;
+  transition: opacity .2s;
+}
+
+.SideBySide__image:hover .SideBySide__caption {
```

Comment:
I'm not super keen on showing things only on hover, but maybe this is okay. Did you consider using a `title` attribute instead?